yaptide/batch/fabric_test.py

```diff
@@ -1,8 +1,18 @@
 from fabric import Connection, Result
+from paramiko import Ed25519Key
 
-login = "---------------"
-host = "ares"
-connection = Connection(host=f'{login}@{host}.cyfronet.pl', connect_kwargs={"password": "---------------"})
+import os
+
+from pathlib import Path
+
+ROOT_DIR = os.path.dirname(os.path.realpath(__file__))
```

Comment:
use pathlib

Reply:
This is for dummy purpose and will be deleted

Reply:
ok, good !